framework/src/materials/GenericConstantMaterial.C

```diff
@@ -46,6 +46,12 @@ GenericConstantMaterial::GenericConstantMaterial(const InputParameters & paramet
 }
 
 void
+GenericConstantMaterial::initQpStatefulProperties()
```

Comment:
Then it should be an error. We are not supporting auto promotion in this PR, arenot we?